### DISCUSSION TITLE
Where were the PPT default found?

### DISCUSSION THREAD

**Discussion by flukejones** [seerge/g-helper#3178] [Q&A]
Hi Sergei, wondering where you found the PPT default values?

**Comment by seerge** [comment]
@flukejones hello, I don't know default PPT values :) I just set sliders to 80W by default in the UI (but they have no effect until `Apply` is clicked of course) :)

You can read SPL, SPPT, FPPT via `ryzenadj` for AMD models

**Reply by flukejones** [reply]
Oh... hmm.. Okay where did you find the min and max values?

**Reply by seerge** [reply]
Min values are just 5w. Max values are based on what people see in AC.
